docs/4_2_co-simulation_api.adoc

```diff
@@ -215,14 +215,14 @@ If the <<earlyReturnTime>> is greater than the current <<intermediateUpdateTime>
 |<<each-state-description,Equations and Actions>>
 |Functions Influencing Equations
 
-|Get intermediate variable values latexmath:[\mathbf{v}_u(\mathbf{t}_u)]
+|Get intermediate variable values latexmath:[\mathbf{v}_u(t_u)]
 |<<get-and-set-variable-values,`fmi3Get{VariableType}`>>
 
 |Get time derivatives of <<continuous>> <<output,`outputs`>> latexmath:[\mathbf{y}_\mathit{c,u}^{(j)}(\mathbf{t_\mathit{u}})].
```

Comment:
I dropped all the italic subscript instructions as they are the default anyway. If the default ever changes, it will consistently deal with formatting now.

Reply:
Please reopen if I misunderstood.